src/main/java/com/cloudbees/plugins/credentials/ViewCredentialsAction.java

```diff
@@ -223,11 +223,14 @@ public String getStoreBaseUrl(String itUrl) {
      * @return {@code true} if the action should be visible.
      */
     public boolean isVisible() {
-        if (context instanceof AccessControlled
-                && !((AccessControlled) context).hasPermission(CredentialsProvider.VIEW)) {
-            // must have permission
-            return false;
+        if (context instanceof AccessControlled) {
+            AccessControlled accessControlled = (AccessControlled) this.context;
+            if (accessControlled.hasPermission(Jenkins.ADMINISTER) || !accessControlled.hasPermission(CredentialsProvider.VIEW)) {
+                // must have permission, Administrator's view credentials from 'Manage Jenkins'
+                return false;
```

Comment:
FWIW I found this (b2de99c0df2f8b775d9d418f0a6ddbc6fb36603b) very confusing and reported it as a bug for CloudBees CI until I tracked this down in history: by _gaining_ the Overall/Administer permission, a sidebar link is suddenly hidden from you, which is the opposite of what you would normally expect.